arch/arm/include/armv7-a/irq.h

```diff
@@ -36,6 +36,9 @@
 #  include <stdint.h>
 #endif
 
+#include "cp15.h"
```

Comment:
```suggestion
#include <arch/cp15.h>
```

Reply:
change to:
```
#include <arch/armv7-a/cp15.h>
```
the standard header file should be included through `<>`

Reply:
Done